.travis.yml

```diff
@@ -1,23 +1,81 @@
-os:
-  - linux
-  - osx
+language: c
 
-# This makes us use Ubuntu 14 instead of 12
-dist: trusty
+matrix:
+  include:
+    - env:
+      - BUILD_TYPE: shared-linux-amd64
```

Comment:
And yes, for consistency with prior releases, this should be `linux-amd64-shared` (and likewise for the rest), but again, I'm not sure if github or something else is imposing a requirement on these names. Are they?

Reply:
> Are they?

No, you can call them as you prefer...

Reply:
I already refactored this :)

Reply:
Well, sorry - only appveyor was refactored for now.